packages/collector/test/agentCommunication_test.js

```diff
@@ -67,8 +67,8 @@ describe('agentCommunication', function () {
         )
       ));
 
-  it('sends a Node.js EOL alert event to the agent (when applicable)', async () => {
-    return retry(async () => {
+  it('sends a Node.js EOL alert event to the agent (when applicable)', async () =>
+    retry(async () => {
```

Comment:
Question: Is this rule enforced now? That would be not ideal. I'd like to make this optional.

Reply:
Yes, that is enforced now. I don't have a strong opinion, I'll set the rule `off` again.

Reply:
🙏